### ISSUE TITLE
[v2.0.0-alpha.8] Error in migrating to the v2.0.0-alpha.8 version

### ISSUE THREAD

**Issue by codingkarthik** [hasura/graphql-engine#6802]
While migrating to the `v2.0.0-alpha.8` version using a database, the following error is thrown:

```
{"type":"startup","timestamp":"2021-04-20T18:27:40.313+0000","level":"info","detail":{"kind":"postgres_connection","info":{"retries":1,"database_url":"postgres://karthikeyan:..........@localhost:5432/hasura_2_7"}}}
{"type":"startup","timestamp":"2021-04-20T18:27:40.313+0000","level":"error","detail":{"kind":"catalog_migrate","info":{"path":"$","error":"Cannot use database previously used with a newer version of graphql-engine (expected a catalog version <=46, but the current version is 45).","code":"not-supported"}}}
```

Switching back to `v2.0.0-alpha.7` will work.

You can also manually run the following and then start `v2.0.0-alpha.8`

```
BEGIN;

CREATE UNIQUE INDEX hdb_cron_events_unique_scheduled
ON hdb_catalog.hdb_cron_events (trigger_name, scheduled_time)
WHERE status = 'scheduled';

INSERT INTO hdb_catalog.hdb_version (version, upgraded_on) VALUES (46, NOW())
ON CONFLICT ((version IS NOT NULL))
DO UPDATE SET version = 46, upgraded_on = NOW();

COMMIT;
```

**Comment by nasatome** [comment]
Same problem, in  Operating System: Manjaro Linux, Docker Server Version: 20.10.6 

migrating from v2.0.0-alpha.7 to v2.0.0-alpha.8



```
graphql-engine:
    image: hasura/graphql-engine:v2.0.0-alpha.7 # -> hasura/graphql-engine:v2.0.0-alpha.8
    ports:
      - "127.0.0.1:8080:8080"
    depends_on:
      - postgres
    restart: always
    networks:
      - default_data_net
    environment:
      HASURA_GRAPHQL_DATABASE_URL: postgres://${POSTGRES_USER}:${POSTGRES_PASSWORD}@postgres:5432/${POSTGRES_DB}
      HASURA_GRAPHQL_ENABLE_CONSOLE: ${HASURA_GRAPHQL_ENABLE_CONSOLE} 
      HASURA_GRAPHQL_DEV_MODE: ${HASURA_GRAPHQL_DEV_MODE}
      HASURA_GRAPHQL_ENABLED_LOG_TYPES: startup, http-log, webhook-log, websocket-log, query-log
      HASURA_GRAPHQL_ADMIN_SECRET: ${HASURA_GRAPHQL_ADMIN_SECRET}
      HASURA_GRAPHQL_JWT_SECRET: ${HASURA_GRAPHQL_JWT_SECRET}
```

**Comment by tirumaraiselvan** [comment]
Please run the below on the metadata database and then restart hasura. We will release a new version with the fix ASAP.

```
BEGIN;

CREATE UNIQUE INDEX hdb_cron_events_unique_scheduled
ON hdb_catalog.hdb_cron_events (trigger_name, scheduled_time)
WHERE status = 'scheduled';

INSERT INTO hdb_catalog.hdb_version (version, upgraded_on) VALUES (46, NOW())
ON CONFLICT ((version IS NOT NULL))
DO UPDATE SET version = 46, upgraded_on = NOW();

COMMIT;
```

**Comment by tirumaraiselvan** [comment]
Fixed in `v2.0.0-alpha.9`
